RecessApplication/models.py

```diff
@@ -16,12 +16,14 @@ class CustomUser(AbstractBaseUser):
     physical_id_num = models.CharField(max_length=100, blank=True, default='')
     dob = models.DateField(auto_now_add=False)
     role = models.CharField(max_length=100, blank=True, default='')
+    photo = models.CharField(max_length=1000000, blank=True, default='')
 
     # Password included already...
     #password = models.CharField(max_length=100, blank=True, default='')
 
     USERNAME_FIELD = 'email_address'
-    REQUIRED_FIELDS = ['first_name', 'last_name','preferred_name','physical_id_num','dob','role']
+#    REQUIRED_FIELDS = ['first_name', 'last_name','preferred_name','physical_id_num','dob','role']
```

Comment:
do we need this line still or can we just remove it instead of commenting it out?